pkg/util/uitl.go

```diff
@@ -0,0 +1,45 @@
+package util
+
+import (
+	"fmt"
+	"github.com/pg-sharding/spqr/pkg/models/distributions"
+	"github.com/pg-sharding/spqr/pkg/models/kr"
+	"strings"
+)
+
+// GetKRCondition returns SQL condition for elements of distributed relation between two key ranges
+// TODO support multidimensional key ranges
+func GetKRCondition(ds *distributions.Distribution, rel *distributions.DistributedRelation, kRange *kr.KeyRange, upperBound kr.KeyRangeBound, prefix string) string {
```

Comment:
I don't understand two things here

1. What this method do and what `KRConditions` is.
2. Why it is in `pkg/util/uitl.go`? It looks like not an util method.

btw in filename there is a typo

Reply:
What this method does is described in comment above. Also I moved it to kr package